native.js

```diff
@@ -2,7 +2,21 @@ const { existsSync } = require('node:fs');
 const path = require('node:path');
 const { platform, arch, report } = require('node:process');
 
-const isMusl = () => !report.getReport().header.glibcVersionRuntime;
+const isMusl = () => {
+  try {
+    return !report.getReport().header.glibcVersionRuntime;
+  } catch (_e) {
```

Comment:
JavaScript does indeed support catch without argument [since Node 10](https://node.green/#ES2019-misc-optional-catch-binding)


```suggestion
  } catch {
```